docs/02.API-REFERENCE.md

```diff
@@ -166,11 +166,19 @@ RegExp object optional flags:
 
   - JERRY_REGEXP_FLAG_GLOBAL - global match; find all matches rather than stopping after the first match
   - JERRY_REGEXP_FLAG_IGNORE_CASE - ignore case
-  - JERRY_REGEXP_FLAG_MULTILINE - multiline; treat beginning and end characters (^ and $) as working over
-  multiple lines (i.e., match the beginning or end of each line (delimited by \n or \r), not only the
-  very beginning or end of the whole input string)
+  - JERRY_REGEXP_FLAG_MULTILINE - multiline; treat beginning and end characters (^ and $) as working
+    over
+    multiple lines (i.e., match the beginning or end of each line (delimited by \n or \r), not only the
+    very beginning or end of the whole input string)
+  - JERRY_REGEXP_FLAG_STICKY - The sticky flag indicates that it matches only from the index indicated
+    by the lastIndex property
+  - JERRY_REGEXP_FLAG_UNICODE - The unicode flag enables various Unicode-related features
+  - JERRY_REGEXP_FLAG_DOTALL -The dotall flag indicates that the dot special character (".") should
+    additionally match the following line terminator ("newline") characters in a string;
+
+
+*Changed in version[[NEXT_RELEASE]]* : Added `JERRY_REGEXP_FLAG_STICKY`, `JERRY_REGEXP_FLAG_UNICODE` , `JERRY_REGEXP_FLAG_DOTALL` values.
 
-*New in version 2.0*.
```

Comment:
Please don't remove the "New in version" notices, just add the "Change in version.." line. We want to keep the information on when was this enum introduced.